package.json

```diff
@@ -137,6 +137,7 @@
   },
   "dependencies": {
     "@babel/polyfill": "^7.0.0",
+    "@clusterws/cws": "^0.10.0",
```

Comment:
same as uws, but kept updated by the community

Reply:
👍